core/src/main/scala/org/bitcoins/core/protocol/dlc/ContractDescriptorTemplate.scala

```diff
@@ -0,0 +1,138 @@
+package org.bitcoins.core.protocol.dlc
+
+import org.bitcoins.core.currency._
+
+sealed trait ContractDescriptorTemplate {
```

Comment:
require(individualCollateral < totalCollateral, s"Cannot have individual collateral be greater than totalCollateral, individualCollateral=$individualCollateral, totalCollateral=$totalCollateral")